pylana/filter.py

```diff
@@ -0,0 +1,193 @@
+import time
+from dateutil.parser import parse
+
+def combine_filters(*filters) -> list:
+    """
+    Create a list of combined filters.
+
+    Args:
+        *filters:
+            Filter arguments to be combined into a trace filter sequence. They
+            can be either a single filter as a dictionary or multiple filters in
+            a list.
+
+    Returns:
+        A list containing the combined filters.
+    """
+    combined_filter = []
+
+    for trace_filter in filters:
+        if type(trace_filter) is list:
+            combined_filter += trace_filter
+        else:
+            combined_filter.append(trace_filter)
+
+    return(combined_filter)
+
+
+def create_timespan_filter(start: str, end: str) -> dict:
+    """
+    Create a timespan filter to be used in a trace filter sequence.
+
+    Args:
+        start:
+            A string denoting the start timestamp.
+        end:
+            A string denoting the end timestamp.
+
+    Returns:
+        A dictionary containing the filter.
+    """
+    timespan_filter = {'startInRange': False,
+                       'endInRange': False,
+                       'type': 'timeRangeFilter',
+                       'from': time.mktime(parse(start).timetuple()) * 1000,
+                       'to': time.mktime(parse(end).timetuple()) * 1000}
```

Comment:
The function `parse` returns a `datetime` object. It offers a method `timestamp`, which returns the seconds passed since epoch. This simplifies the timestamp generation to `parse(start).timestamp() * 1000` and same for `end`.